block/src/lib.rs

```diff
@@ -67,6 +67,9 @@ use vmm_sys_util::{ioctl_io_nr, ioctl_ioc_nr};
 const SECTOR_SHIFT: u8 = 9;
 pub const SECTOR_SIZE: u64 = 0x01 << SECTOR_SHIFT;
 
+/// Nesting depth limit for disk formats that can open other disk files.
+pub const MAX_NESTING_DEPTH: u32 = 10;
```

Comment:
10? I think 3 sounds like a reasonable maximum.

Reply:
I think 3 is to little. For example, the backing file of qcow2 behaves like overlayfs and it can be used as container image for secure containers based on virtual machines.